adapter/redisquota/driver.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package redisquota
+
+// ConnPool interface for a redis connection pool.
+type ConnPool interface {
```

Comment:
In general, all symbols declared in the redisquota adapter should be internal (start with a lowercase), except those explicitly required (like NewAspect & friends).

So please change to connPool